library/std/src/sys/pal/windows/c/windows_targets.rs

```diff
@@ -0,0 +1,15 @@
+pub macro link {
+    ($library:literal $abi:literal $($link_name:literal)? $(#[$doc:meta])? fn $($function:tt)*) => (
+        #[link(name = "kernel32")]
```

Comment:
Note: windows-bindgen links to a pre-built `windows.lib` which we don't have in the rust repo. So instead we always link to `kernel32.lib` and then list the other import libraries below. This works because `extern` blocks are not connected to the library given in the `#[link]` attribute.

Reply:
should this be in the file as documentation? should this file be documented more in general?

Reply:
Sure, I can add a comment or two.